compose.yaml

```diff
@@ -0,0 +1,8 @@
+services:
```

Comment:
we don't need compose in this project, please remove this

one can do anything with docker run